src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -25,7 +25,9 @@ import no.skatteetaten.aurora.boober.service.CantusService
 @Service
 class AzureFeature(
     cantusService: CantusService,
-    @Value("\${clinger.sidecar.default.version:0.4.0}") val sidecarVersion: String
+    @Value("\${clinger.sidecar.default.version:0.4.0}") val sidecarVersion: String,
+    @Value("\${clinger.sidecar.default.ldapurl:ldap://ldap.skead.no:389}") val defaultLdapUrl: String,
```

Comment:
Ville ikke hatt default her heller, egentlig... Slike urls er jo kanskje ikke noe vi vil ha på en public github?

Reply:
Jeg kan legge inn "tom default" - `@Value("\${clinger.sidecar.default.ldapurl:}")` . Det vil medføre at boober starter opp uten denne konfig, men vil lage clinger-versjoner som ikke fungerer. Det er rimelig, ettersom dette egentlig er intern funksjonalitet for skatt.